qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -238,30 +412,25 @@ def _do_get_device_type(self):
 
     ########################################################################
 
-    def initialisation(self, flag=0):
-        mode = self.get('device_mode')
+    def _initialise(self, flag: int=0) -> None:
```

Comment:
what are the possible flags? Does it need to be documented?

Reply:
I think it should just be removed. This is what the manual has to say
```
This value is currently unused. Pass 0 as a parameter. 
```